notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-wav2vec2.ipynb

```diff
@@ -8,12 +8,12 @@
    },
```

Comment:
<p>Compare Accuracy of the original PyTorch, original OpenVINO and Quantized Models -&gt; Compare Accuracy of original PyTorch model, OpenVINO FP16 and INT8 models</p><p> </p><p>This sounds more clear</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1059/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>